### PR TITLE
add icons tutorial

### PR DESCRIPTION
**Pull request by mattgreenfield**
A WDK user in support has had trouble adding the social footer icons so I've written a tutorial on using icons in general.

@AidanThreadgold 
@benjaminparry

### CODE REVIEW

File: _tutorials/icons.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: side-nav
+title: Using Icons
+last-updated: 17-08-2015
+---
+
+We use icon fonts to display icons.
+
+To display icons on your WDK template you will need to place the following tags in the templates `<head>`;
```
Comment (Dayjo) [inline _tutorials/icons.md]: `template's` ?

**Comment by Dayjo** [comment]
Looks good Matt

File: _tutorials/icons.md (hunk 1 above)
Comment (mattgreenfield) [inline _tutorials/icons.md]: which one?

**Comment by benjaminparry** [comment]
Nice one Matt. I thought there was an issue relating to this but I couldn’t find one.

**Comment by benjaminparry** [comment]
We should merge this

File: _tutorials/icons.md (hunk 1 above)
Comment (Dayjo) [inline _tutorials/icons.md]: "in the templates `<head>`" should be; "in the template's `<head>`"

**Comment by mattgreenfield** [comment]
Yes, I'll add it to my list. Cheers

**Pull request merged**